src/cli/help_menu.js

```diff
@@ -0,0 +1,27 @@
+export function getHelpMenu() {
+    console.log("\n\n", "Options:", "\n\n\n",
+    "Flag","\n\n",
+    "-", "\t\t\t\t", "script read from stdin","\n",
```

Comment:
deltachat-desktop does not support user scripts at this time

Reply:
ok I changed it, thanks !